src/Components/RightSidebar.elm

```diff
@@ -196,14 +198,82 @@ localTab =
 
 
-audioTab : Element msg
-audioTab =
-    el [ width fill, height fill, Background.color Colors.playlistBackground, padding 8 ] (text "Audio Tab")
+audioTab : Maybe ItemDetails -> Bool -> Element msg
+audioTab currentlyPlaying playing =
+    case currentlyPlaying of
+        Nothing ->
+            Element.none
 
+        Just item ->
+            case item.mediatype of
+                WSDecoder.Audio ->
+                    row [ width fill, height (px 80), Background.color Colors.black ]
+                        [ case playing of
+                            True ->
+                                Element.image [ alignRight, height (px 30), width (px 30), alignLeft, paddingXY 20 0 ]
+                                    { description = ""
+                                    , src = "wave.gif"
+                                    }
 
-videoTab : Element msg
-videoTab =
-    el [ width fill, height fill, Background.color Colors.playlistBackground, padding 8 ] (text "Video Tab")
+                            False ->
+                                el [ alignRight, height (px 30), width (px 30), alignLeft, paddingXY 20 0, Font.color white, Font.size 20 ] (text "---")
+                        , column [ padding 20, spacing 10 ]
+                            [ row [] [ el [ Font.color white ] (text item.title) ]
+                            , row []
+                                (List.map
+                                    (\artist ->
+                                        el [ Font.color (Element.rgb 0.6 0.6 0.6), Font.size 13 ] (text artist)
+                                    )
+                                    item.artist
+                                )
+                            ]
+                        , Element.image [ alignRight, height (px 70), width (px 70) ]
+                            { description = ""
+                            , src = crossOrigin "http://localhost:8080" [ "image", percentEncode item.thumbnail ] []
+                            }
+                        ]
+
+                WSDecoder.Video ->
+                    Element.none
+
+
+videoTab : Maybe ItemDetails -> Bool -> Element msg
+videoTab currentlyPlaying playing =
+    case currentlyPlaying of
+        Nothing ->
+            Element.none
+
+        Just item ->
+            case item.mediatype of
+                WSDecoder.Audio ->
+                    Element.none
+
+                WSDecoder.Video ->
+                    row [ width fill, height (px 80), Background.color Colors.black ]
+                        [ case playing of
+                            True ->
+                                Element.image [ alignRight, height (px 30), width (px 30), alignLeft, paddingXY 20 0 ]
+                                    { description = ""
+                                    , src = "wave.gif"
+                                    }
+
+                            False ->
```

Comment:
Is this even possible? Can `currentlyPlaying` be `Just` when `playing` is `false`? Or will curretlyPlaying still be filled for paused items?

Reply:
The boolean value of the `playing` variable indicates whether the media is playing or has paused. Even when the item has paused, we continue to receive JSON data from the WebSocket, and `currentlyPlaying`will continue to be filled until the item has completely stopped playing. So, yes it is possible that `currentlyPlaying` can be `Just` when `playing` is `false`.